tsdb/head_append.go

```diff
@@ -1962,6 +1965,6 @@ func (a *headAppender) Rollback() (err error) {
 
 	// Series are created in the head memory regardless of rollback.
 	// Thus we have to commit them in the postings and to log them to the WAL in any case.
-	a.head.postings.Commit()
+	a.head.postings.Commit(storage.SeriesRef(a.highestCreatedSeriesRef))
```

Comment:
Could `a.highestCreatedSeriesRef` change during the execution of this method?

Reply:
It shouldn't, it's invalid to call `Append()` and `Commit()` concurrently.